plugins/att_cybersecurity_alienvault_otx/komand_att_cybersecurity_alienvault_otx/connection/connection.py

```diff
@@ -1,27 +1,22 @@
-import komand
+import insightconnect_plugin_runtime
 from .schema import ConnectionSchema, Input
 
 # Custom imports below
 from OTXv2 import OTXv2
+from insightconnect_plugin_runtime.exceptions import PluginException, ConnectionTestException
 
 
-class Connection(komand.Connection):
+class Connection(insightconnect_plugin_runtime.Connection):
     def __init__(self):
         super(self.__class__, self).__init__(input=ConnectionSchema())
         self.client = None
 
-    def connect(self, params):
-        api_key, otx_server = (
-            params.get(Input.API_KEY).get("secretKey", None),
-            params.get(Input.URL),
-        )
-        try:
-            self.client = OTXv2(api_key, server=otx_server)
-        except Exception as e:
-            raise Exception(f"An error has occurred while connecting: {e}")
+    def connect(self, params={}):
+        self.client = OTXv2(params.get(Input.API_KEY).get("secretKey"), server=params.get(Input.URL))
```

Comment:
Same as above. I would put the creation of that object into `try-except` in order to catch any errors if they occur

Reply:
I improved error handling by catching all exceptions from OTXv2 and raising them again using preset equivalents from ConnectionTestException.